WalletWasabi/Crypto/Slip21Node.cs

```diff
@@ -11,23 +11,25 @@ public class Slip21Node
 		private static readonly int KEY_SIZE = 32;
 		private byte[] _data;
 
-		public Slip21Node(byte[] data) =>
+		public Slip21Node(byte[] data)
+		{
```

Comment:
The title says "Use expression body..." but here you are eliminating the expression body and replacing it by a "normal" body. Anyway, what's the point?

Reply:
I meant the opposite "Do not use expression body" and "use block body", because that is what we use for the constructors throughout the code.